test/object-store/sync/flx_sync.cpp

```diff
@@ -2916,17 +2956,26 @@ TEST_CASE("flx: data ingest", "[sync][flx][data ingest][baas]") {
 
             auto table = realm->read_group().get_table("class_Asymmetric");
             REQUIRE(table->size() == 0);
+
+            auto documents = get_documents(*realm->config().sync_config->user, "Asymmetric", 100);
+            for (int i = 0; i < 100; ++i) {
```

Comment:
nit: use the container size (i.e, obj_ids.size()) (+ in some other places below)